assets/dev/js/editor/components/documents/commands/internal/attach-preview.js

```diff
@@ -34,7 +46,11 @@ export class AttachPreview extends $e.modules.CommandInternalBase {
 				return resolve();
 			}
 
-			document.$element = elementor.$previewContents.find( '.elementor-' + document.id );
+			if ( selector?.length ) {
+				document.$element = elementor.$previewContents.find( selector );
+			} else {
+				document.$element = elementor.$previewContents.find( '.elementor-' + document.id );
+			}
```

Comment:
```suggestion
			selector ||= '.elementor-' + document.id;
			document.$element = elementor.$previewContents.find(  );
			
```

Reply:
Parsing error: This experimental syntax requires enabling the parser plugin: 'logicalAssignment'

Reply:
Just add it

Reply:
Handled other way:)